bedrock/newsletter/templates/newsletter/existing.html

```diff
@@ -114,7 +114,6 @@ <h1>{{ _('Manage your Email Preferences') }}</h1>
           <tr>
             <th></th>
             <th>{{ _('Subscribe') }}</th>
-            <th>{{ _('Unsubscribe') }}</th>
```

Comment:
There are still 3 columns to this table. Do we need a `colspan=2` on the `<th>` above?

Reply:
Good catch 👍